docs/dashboard-config.md

```diff
@@ -34,6 +34,7 @@ The following are a list of features that are supported, along with there defaul
 | disablePerformanceMetrics    | false   | Disables Endpoint Performance tab from Model Serving metrics.                                        |
 | disableDistributedWorkloads  | false   | Disables Distributed Workload Metrics from the dashboard.                                            |
 | disableModelRegistry         | true    | Disables Model Registry from the dashboard.                                                          |
+| disableConnectionTypes       | true    | Disables creating custom data connection types from the dashboard.                                   |
 
 ## Defaults
 
```

Comment:
I'm not sure if we are trying to keep all the options in the sections below for the `Defaults` and the `Example OdhDashboard Config` sections. If so, we should add the `disableConnectionTypes` field there as well.

Reply:
true, yeah. So i see `disableModelRegistry` is only in the table. 

But I'll add `disableConnectionTypes` to the yaml example